tests/golden/vshn/appcat/appcat/21_composition_vshn_postgres.yaml

```diff
@@ -621,6 +632,12 @@ spec:
         - fromConnectionSecretKey: POSTGRESQL_PASSWORD
           name: POSTGRESQL_PASSWORD
           type: FromConnectionSecretKey
+        - fromConnectionSecretKey: PRIMARY_IP
```

Comment:
Is this actually necessary? You already export the values as a connectiondetail in the compostion function.